dist/index.html

```diff
@@ -6,6 +6,7 @@
 	<title>Restuarant Page</title>
 </head>
 <body>
-	 <script src="main.js"></script>
+	 <div id="content"></div>
+	  <script src="bundle.js"></script>
```

Comment:
it should be main.js not bundle.js [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/dist_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)